commit 13794d0e9f7e7483d68b85ad8247c824f0c9c2ed (refs/changes/18/3018/3)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2017-12-16T08:11:28+01:00 (1 year, 10 months ago)
    
    HEAD-2378 Add dns_domain to sapi metadata
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/scripts/headnode.sh b/scripts/headnode.sh
index ce58f72b..b34f7e44 100755
--- a/scripts/headnode.sh
+++ b/scripts/headnode.sh
@@ -639,6 +639,12 @@ download_metadata
 write_initial_config
 registrar_setup
 HERE
+
+        if [[ -n "${CONFIG_dns_domain}" ]]; then
+            vmadm update $sapi_uuid <<EOF
+  {"set_customer_metadata": {"dns_domain": "${CONFIG_dns_domain}"}}
+EOF
+        fi
         setup_state_add "sapi_bootstrapped"
     fi
 }
